### PR TITLE
Create ServicePayload definition

### PR DESCRIPTION
**Pull request by francescopersico**
Split Service definition in:

ServicePayload: used to create/update service in services function
Service: used as return type in get/create/update in services function, used as create/update type in admin function

### CODE REVIEW

File: openapi/definitions.yaml [hunk 1]
```diff
@@ -527,18 +527,18 @@ SenderMetadata:
     - service_name
     - organization_name
     - department_name
-Service:
-  description: A Service tied to an user's subscription.
+ServicePayload:
+  description: A payload used to create/update a service for a user.
```
Comment (gunzip) [inline openapi/definitions.yaml:531]: why remove service_id ? (we are used to create a service provinding its id)

Comment (francescopersico) [reply openapi/definitions.yaml:531]: If needed you can use the Service type

**Comment by gunzip** [comment]
is this mergeable? lgtm

**Review by AleDore — COMMENTED** [review]
lgtm. `Service` definition is unchanged.

**Pull request merged**